src/app/sharehouses/[share_house_id]/page.tsx

```diff
@@ -17,119 +19,58 @@ export interface IShareHousePageProps {
   };
 }
 
-const ShareHousePage = ({
-  params: {
-    share_house_id,
-    shareHouseName,
-    currentStartDate,
-    currentEndDate,
-    progressPercent,
-    nextStartDate,
-    nextEndDate,
-    cardContentCurrent,
-    cardContentNext,
-  },
+const ShareHousePage = async ({
+  params: { share_house_id, shareHouseName },
 }: IShareHousePageProps) => {
-  console.log('⭐️ShareHousePage ID:', share_house_id);
-  console.log('⭐️ShareHousePage Name:', shareHouseName);
-
-  currentStartDate = '2022-01-01';
-  currentEndDate = '2022-01-07';
-  progressPercent = 50;
-  nextStartDate = '2022-01-08';
-  nextEndDate = '2022-01-14';
-
-  // EXAMPLE: TENANTS
-  cardContentCurrent = [
-    {
-      category: null,
-      isComplete: false,
-      taskNum: 0,
-      completedTaskNum: 0,
-      tenant: 'Akio',
-    },
+  const res = await api.rotation[':shareHouseId'].$get(
+    { param: { shareHouseId: share_house_id } },
     {
-      category: 'Kitchen',
-      tenant: 'Matio',
-      isComplete: true,
-      taskNum: 4,
-      completedTaskNum: 4,
+      headers: {
+        cookie: headers().get('cookie') || '', // Add cookies to headers
+      },
     },
-    {
-      category: 'Bathroom',
-      tenant: 'Akio',
-      isComplete: false,
-      taskNum: 9,
-      completedTaskNum: 3,
-    },
-    {
-      category: 'Living room',
-      tenant: 'Maaaatio',
-      isComplete: false,
-      taskNum: 7,
-      completedTaskNum: 2,
-    },
-  ];
+  );
+  // Convert response to JSON
+  const data = await res.json();
 
-  // // EXAMPLE: NO TENANTS
-  // cardContentCurrent = [
-  //   {
-  //     category: null,
-  //   }
-  // ];
+  // Check for error in data and display it if found
+  if ('error' in data) {
+    return <div>{data.error}</div>;
+  }
 
-  cardContentNext = [
-    {
-      category: null,
-      tenant: 'Matio',
-      taskNum: 4,
-    },
-    {
-      category: 'Bathroom',
-      tenant: 'Akio',
-      taskNum: 9,
-    },
-    {
-      category: 'Living room',
-      tenant: 'Maaaatio',
-      taskNum: 7,
-    },
-  ];
+  const formatDate = (date: string) => {
```

Comment:
Could you use this function to formate dates?
https://github.com/Tascurator/tascurator-frontend/blob/5273ed95d352a327f50bbff950b212b75db3699a/src/utils/dates.ts#L90

Reply:
Also, could you convert the date to PDT?

Reply:
I imported the `formatDate` method and converted the date to PDT.